````
Caused by: org.apache.maven.plugin.MojoFailureException: File '/home/runner/work/quarkus/quarkus/integration-tests/gradle/src/test/java/io/quarkus/gradle/AddExtensionToSingleModuleKtsProjectTest.java' has not been previously formatted.  Please format file and commit before running validation!
````

--- a/integration-tests/gradle/src/test/java/io/quarkus/gradle/AddExtensionToModuleInMultiModuleKtsProjectTest.java
+++ b/integration-tests/gradle/src/test/java/io/quarkus/gradle/AddExtensionToModuleInMultiModuleKtsProjectTest.java
@@ -20,7 +20,7 @@ public class AddExtensionToModuleInMultiModuleKtsProjectTest extends QuarkusGrad
 
         final File projectDir = getProjectDir("add-extension-multi-module-kts");
 
-        runGradleWrapper(projectDir, ":application:addExtension", "--extensions=hibernate-orm");
+        runGradleWrapper(projectDir, ":application:addExtension", "--extensions=openshift");
 
         final Path applicationLib = projectDir.toPath().resolve("application").resolve("settings.gradle");
         assertThat(applicationLib).doesNotExist();
@@ -32,10 +32,10 @@ public class AddExtensionToModuleInMultiModuleKtsProjectTest extends QuarkusGrad
 
         final Path appBuildKts = projectDir.toPath().resolve("application").resolve("build.gradle.kts");
         assertThat(appBuildKts).exists();
-        assertThat(readFile(appBuildKts)).contains("implementation(\"io.quarkus:quarkus-hibernate-orm\")");
+        assertThat(readFile(appBuildKts)).contains("implementation(\"io.quarkus:quarkus-openshift\")");
 
-        runGradleWrapper(projectDir, ":application:removeExtension", "--extensions=hibernate-orm");
-        assertThat(readFile(appBuildKts)).doesNotContain("implementation(\"io.quarkus:quarkus-hibernate-orm\")");
+        runGradleWrapper(projectDir, ":application:removeExtension", "--extensions=openshift");
+        assertThat(readFile(appBuildKts)).doesNotContain("implementation(\"io.quarkus:quarkus-openshift\")");
     }
 
     private static String readFile(Path file) throws IOException {

--- a/integration-tests/gradle/src/test/java/io/quarkus/gradle/AddExtensionToModuleInMultiModuleProjectTest.java
+++ b/integration-tests/gradle/src/test/java/io/quarkus/gradle/AddExtensionToModuleInMultiModuleProjectTest.java
@@ -20,17 +20,17 @@ public class AddExtensionToModuleInMultiModuleProjectTest extends QuarkusGradleW
 
         final File projectDir = getProjectDir("add-extension-multi-module");
 
-        runGradleWrapper(projectDir, ":application:addExtension", "--extensions=hibernate-orm");
+        runGradleWrapper(projectDir, ":application:addExtension", "--extensions=openshift");
 
         final Path applicationLib = projectDir.toPath().resolve("application").resolve("settings.gradle");
         assertThat(applicationLib).doesNotExist();
 
         final Path appBuild = projectDir.toPath().resolve("application").resolve("build.gradle");
         assertThat(appBuild).exists();
-        assertThat(readFile(appBuild)).contains("implementation 'io.quarkus:quarkus-hibernate-orm'");
+        assertThat(readFile(appBuild)).contains("implementation 'io.quarkus:quarkus-openshift'");
 
-        runGradleWrapper(projectDir, ":application:removeExtension", "--extensions=hibernate-orm");
-        assertThat(readFile(appBuild)).doesNotContain("implementation 'io.quarkus:quarkus-hibernate-orm'");
+        runGradleWrapper(projectDir, ":application:removeExtension", "--extensions=openshift");
+        assertThat(readFile(appBuild)).doesNotContain("implementation 'io.quarkus:quarkus-openshift'");
     }
 
     private static String readFile(Path file) throws IOException {

--- a/integration-tests/gradle/src/test/java/io/quarkus/gradle/AddExtensionToSingleModuleKtsProjectTest.java
+++ b/integration-tests/gradle/src/test/java/io/quarkus/gradle/AddExtensionToSingleModuleKtsProjectTest.java
@@ -17,18 +17,19 @@ public class AddExtensionToSingleModuleKtsProjectTest extends QuarkusGradleWrapp
 
         final File projectDir = getProjectDir("add-remove-extension-single-module-kts");
 
-        runGradleWrapper(projectDir, ":addExtension", "--extensions=hibernate-orm");
+        runGradleWrapper(projectDir, ":addExtension", "--extensions=openshift");
 
         final Path buildKts = projectDir.toPath().resolve("build.gradle.kts");
         assertThat(buildKts).exists();
         assertThat(new String(Files.readAllBytes(buildKts)))
-                .contains("implementation(\"io.quarkus:quarkus-hibernate-orm\")")
+                .contains("implementation(\"io.quarkus:quarkus-openshift\")")
                 .doesNotContain("implementation(enforcedPlatform(\"io.quarkus:quarkus-bom:")
                 .doesNotContain("implementation(\"io.quarkus:quarkus-bom:");
 
-        runGradleWrapper(projectDir, ":removeExtension", "--extensions=hibernate-orm");
+        runGradleWrapper(projectDir, ":removeExtension", "--extensions=openshift");
+
         assertThat(new String(Files.readAllBytes(buildKts)))
-                .doesNotContain("implementation(\"io.quarkus:quarkus-hibernate-orm\")");
+                .doesNotContain("implementation(\"io.quarkus:quarkus-openshift\")");
 
         assertThat(projectDir.toPath().resolve("build.gradle")).doesNotExist();
     }

--- a/integration-tests/gradle/src/test/java/io/quarkus/gradle/AddExtensionToSingleModuleProjectTest.java
+++ b/integration-tests/gradle/src/test/java/io/quarkus/gradle/AddExtensionToSingleModuleProjectTest.java
@@ -17,17 +17,17 @@ public class AddExtensionToSingleModuleProjectTest extends QuarkusGradleWrapperT
 
         final File projectDir = getProjectDir("add-remove-extension-single-module");
 
-        runGradleWrapper(projectDir, ":addExtension", "--extensions=hibernate-orm");
+        runGradleWrapper(projectDir, ":addExtension", "--extensions=openshift");
 
         final Path build = projectDir.toPath().resolve("build.gradle");
         assertThat(build).exists();
         assertThat(new String(Files.readAllBytes(build)))
-                .contains("implementation 'io.quarkus:quarkus-hibernate-orm'")
+                .contains("implementation 'io.quarkus:quarkus-openshift'")
                 .doesNotContain("implementation enforcedPlatform('io.quarkus:quarkus-bom:")
                 .doesNotContain("implementation 'io.quarkus:quarkus-bom:");
 
-        runGradleWrapper(projectDir, ":removeExtension", "--extensions=hibernate-orm");
-        assertThat(new String(Files.readAllBytes(build))).doesNotContain("implementation 'io.quarkus:quarkus-hibernate-orm'");
+        runGradleWrapper(projectDir, ":removeExtension", "--extensions=openshift");
+        assertThat(new String(Files.readAllBytes(build))).doesNotContain("implementation 'io.quarkus:quarkus-openshift'");
 
     }
 

